README.md

```diff
@@ -31,6 +31,60 @@ bin/rake
 
 *Note: As uploads use S3, you'll need to add details to the .env file for them to work. Alternatively, [replace S3 with ActiveStorage](https://github.com/forsbergplustwo/partner-metrics/issues/20).*
 
+## S3 - Minimum setup guide
+
+### 1. Sign up for or login to [Amazon AWS](https://aws.amazon.com)
+### 2. Create an S3 Bucket:
+* In the AWS console, search for S3 and click the offering in the drop down menu. 
+* In the S3 Management Console, click the "Create Bucket" button. 
+* In the bucket creation wizard - name it something cool. (eg... "ice-bucket")
```

Comment:
Do you mind if we give the example as "partner-metrics" instead. It leaves a nice little hint for them in AWS so they know what the bucket is used for.

Reply:
Ha - was just having fun there. Definitely